exporter/opentelemetry-exporter-prometheus/pyproject.toml

```diff
@@ -26,15 +26,16 @@ classifiers = [
 ]
 dependencies = [
   "opentelemetry-api ~= 1.12",
-  "opentelemetry-sdk ~= 1.12",
+  # DONOTMERGE: confirm that this will becomes ~= 1.21 in the next release
+  "opentelemetry-sdk ~= 1.21.0.dev",
```

Comment:
Curious what this is?

Reply:
I wanted confirmation from a maintainer, wasn't sure if this would work. Basically the prom exporter needs the latest SDK at atleast the next release

Reply:
@ocelotl 

Got it. So for next release, we will have to simply remember to update `"opentelemetry-sdk ~= 1.21.0.dev",` to `"opentelemetry-sdk ~= 1.21", before release. I do not believe our release script automatically updates `~=` dependencies.

Reply:
OK lmk if there's anything I can do to make it easier